pytorch_lightning/loops/utilities.py

```diff
@@ -25,21 +25,18 @@
 from pytorch_lightning.utilities.apply_func import apply_to_collection
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 from pytorch_lightning.utilities.fetching import AbstractDataFetcher, DataLoaderIterDataFetcher
-from pytorch_lightning.utilities.finite_checks import detect_nan_parameters
 from pytorch_lightning.utilities.signature_utils import is_param_in_hook_signature
 from pytorch_lightning.utilities.types import STEP_OUTPUT
 
 
-def check_finite_loss(model: "pl.LightningModule", loss: torch.Tensor) -> None:
-    """Checks for finite parameters and loss values.
+def check_finite_loss(loss: torch.Tensor) -> None:
```

Comment:
technically this is a breaking change. could we make a separate folder for internal utilities intended for only the trainer or loops to use?

Reply:
how so? this was never introduced in any release. 
and this already is in a separate folder just for loops.